ci-operator/step-registry/csb/deploy-resources/csb-deploy-resources-ref.yaml

```diff
@@ -14,7 +14,7 @@ ref:
   env:
     - name: CSB_RELEASE
       documentation: CSB release used for selecting right image.
-      default: 4.8.3
+      default: 4.10.3
     - name: CSB_PATCH
       documentation: CSB minor release.
-      default: redhat-00001
+      default: redhat-00016
```

Comment:
what does this version represent? The following are the 4.10.3 versions
```
<version>4.10.3.redhat-00019</version>
<version>4.10.3.redhat-00023</version>
```

Reply:
It's a different thing in this case. Just the way the git examples [branch](https://github.com/openshift/release/blob/master/ci-operator/step-registry/csb/deploy-resources/csb-deploy-resources-commands.sh#L581) is finally composed